.github/workflows/playwright-pr.yml

```
@@ -0,0 +1,136 @@
+      if: steps.changed-files.outputs.any_changed == 'true'
+      run: |
+        echo "Running tests for changed files:"
+        echo "${{ steps.changed-files.outputs.all_changed_files }}"
+        npx playwright test ${{ steps.changed-files.outputs.all_changed_files }} --reporter=list
+      env:
+        NEXT_PUBLIC_SUPABASE_URL: ${{ secrets.NEXT_PUBLIC_SUPABASE_URL }}
+        NEXT_PUBLIC_SUPABASE_ANON_KEY: ${{ secrets.NEXT_PUBLIC_SUPABASE_ANON_KEY }}
+        SUPABASE_SERVICE_ROLE_KEY: ${{ secrets.SUPABASE_SERVICE_ROLE_KEY }}
+        ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
+        BASE_URL: 'http://localhost:3000'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)